fitlins/cli/run.py

```diff
@@ -79,9 +79,9 @@ def get_parser():
                              'removed)')
     g_bids.add_argument('-m', '--model', action='store', default='model.json',
                         help='location of BIDS model description (default bids_dir/model.json)')
-    g_bids.add_argument('-p', '--preproc-dir', action='store', default='fmriprep',
-                        help='location of preprocessed data (if relative path, search '
-                             'bids_dir/derivatives, followed by output_dir)')
+    g_bids.add_argument('-d', '--derivatives', action='append',
```

Comment:
```suggestion
    g_bids.add_argument('-d', '--derivatives', action='store', nargs='+',
```

This requires all derivatives to be listed in the same place (`-d /path/a /path/b` will produce `['/path/a', '/path/b']`, while `-d /path/a -d /path/b` will produce `['/path/b']`), but we follow this idiom with `--participant-label`, as well.

You could do `append` and `nargs='+'`, but then you'd get a list of lists, which would need to be concatenated.